h5py/tests/test_vds/test_highlevel_vds.py

```diff
@@ -398,5 +398,49 @@ def test_relative_vds(self):
     def tearDown(self):
         shutil.rmtree(self.tmpdir)
 
+@ut.skipUnless(vds_support,
+               'VDS requires HDF5 >= 1.9.233')
+class VDSUnlimitedTestCase(ut.TestCase):
+
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+        self.path = osp.join(self.tmpdir, "resize.h5")
+        with h5.File(self.path, "w") as f:
+            source_dset = f.create_dataset(
+                "source",
+                data=np.arange(20),
+                shape=(10, 2),
+                maxshape=(None, 2),
+                chunks=(10, 1),
+                fillvalue=-1
+            )
+            self.layout = h5.VirtualLayout((10, 1), np.int, (None, 1))
```

Comment:
```suggestion
            self.layout = h5.VirtualLayout((10, 1), np.int, maxshape=(None, 1))
```

A bit of explicit naming never hurts. :-)